packages/open-next/src/adapters/routing/middleware.ts

```diff
@@ -98,6 +98,7 @@ export async function handleMiddleware(
       type: internalEvent.type,
       statusCode: res.statusCode,
       headers: {
+        ...res.headers,
```

Comment:
`res.headers` is empty at this point, you probably want to use `resHeaders` instead, but you'll need to do it after they've been set here https://github.com/sst/open-next/blob/2014a1b67f09049b0d8b47f47054adebe24c7a77/packages/open-next/src/adapters/routing/middleware.ts#L126-L135

Reply:
Good catch. I used the responseHeaders and that worked for me

Reply:
@relsunkaev I don't think we should send back all the headers, it will include all the AMZ geo data too... We should just return `result.response.headers.get('set-cookie') ?? ''`

Reply:
That's why i think `resHeaders` is fine it's already what you would have get if it wasn't a redirect, we need to allow people to pass the headers they want, maybe they'd want to pass an api key or an auth header